pulse2percept/effectivecurrent2brightness.py

```diff
@@ -372,66 +379,66 @@ def onoffFiltering(movie, n, sig=[.1, .25],amp=[.01, -0.005]):
         img = insertImg(tmpimg, oldimg)
         filtImgOn=np.zeros([img.shape[0], img.shape[1]])
         filtImgOff=np.zeros([img.shape[0], img.shape[1]])
-        
-        for i in range(n.shape[0]): 
-            [x,y] = np.meshgrid(np.linspace(-1,1,n[i]),np.linspace(-1,1,n[i]))   
+
+        for i in range(n.shape[0]):
+            [x,y] = np.meshgrid(np.linspace(-1,1,n[i]),np.linspace(-1,1,n[i]))
             rsq = x**2+y**2
-            dx = x[0,1]-x[0,0]    
+            dx = x[0,1]-x[0,0]
```

Comment:
I know we're hardly enforcing it, but maybe we should start? PEP8 : white-space after comma and after operator. At some point we can also do a more extensive PEP8 cleanup.